csm_test_utils/as_loadbalancer.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import sys
+import time
+
+import requests
+from influx_line_protocol import Metric
+from ocomone.logging import setup_logger
+
+from .common import base_parser, sub_parsers
+
+AS_LOADBALANCER = "as_loadbalancer"
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+
+
+def report(target, telegraf):
+    """Send request and write metrics to telegraf"""
+    target_req = requests.get(target, headers={"Connection": "close"})
+    influx_row = Metric(AS_LOADBALANCER)
+    if target_req.status_code == 200:
+        influx_row.add_tag("state", "connected")
+        influx_row.add_tag("reason", "ok")
+        influx_row.add_value("elapsed", target_req.elapsed.microseconds / 1000)
```

Comment:
Maybe use int value instead of float? E.g. use `//` instead of `/`

Reply:
other scripts using same style

Reply:
should i refactor them also?